src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars__star stars__star-active"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars stars--2">
+      <div class="stars__star stars__star-active"></div>
+      <div class="stars__star stars__star-active"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars stars--3">
+      <div class="stars__star stars__star-active"></div>
+      <div class="stars__star stars__star-active"></div>
+      <div class="stars__star stars__star-active"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars stars--4">
+      <div class="stars__star stars__star-active"></div>
+      <div class="stars__star stars__star-active"></div>
+      <div class="stars__star stars__star-active"></div>
+      <div class="stars__star stars__star-active"></div>
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars stars--5">
+      <div class="stars__star stars__star-active"></div>
+      <div class="stars__star stars__star-active"></div>
+      <div class="stars__star stars__star-active"></div>
+      <div class="stars__star stars__star-active"></div>
+      <div class="stars__star stars__star-active"></div>
```

Comment:
adding `active` class on every active element is pretty repetetive, don't you think?

look at comment in css file to see what I mean ⬇️